.github/workflows/test.yml

```diff
@@ -21,7 +19,7 @@ jobs:
     - uses: actions/checkout@v1
 
     - name: Setup ROS2
-      uses: ros-tooling/setup-ros@0.0.15
+      uses: ros-tooling/setup-ros@0.0.20
```

Comment:
No problem here, but in the future can we try to keep these types of unrelated changes out of other PRs? They can be their own.